src/modules/dashboard/components/ActionsPage/DetailsWidget/index.ts

```diff
@@ -1,3 +1,7 @@
 export { default } from './DetailsWidget';
 export * from './DetailsWidget';
 export { default as DetailsWidgetTeam } from './DetailsWidgetTeam';
+export { default as DetailsWidgetRoles } from './DetailsWidgetRoles';
+export { default as DetailsWidgetSafe } from './DetailsWidgetSafe';
+export { default as DetailsWidgetAddSafe } from './DetailsWidgetSafe/DetailsWidgetAddSafe';
+export { default as DetailsWidgetSafeTransaction } from './DetailsWidgetSafeTransaction/DetailsWidgetSafeTransaction';
```

Comment:
`DetailsWidgetSafeTransaction` should have its own `index.ts` that exports stuff used by external components, like the other `DetailsWidget...` components